test/integration/targets/git/vars/main.yml

```diff
@@ -36,6 +36,9 @@ repo_format3: 'ssh://git@github.com/jimi-c/test_role.git'
 repo_submodules: 'https://github.com/abadger/test_submodules_newer.git'
 repo_submodule1: 'https://github.com/abadger/test_submodules_subm1.git'
 repo_submodule2: 'https://github.com/abadger/test_submodules_subm2.git'
+repo_submodules_b: 'https://github.com/theunkn0wn1/ansible_test_submodules_monorepo.git'
+repo_submodule3: 'https://github.com/theunkn0wn1/ansible_test_submodules_subm3.git'
+repo_submodule4: 'https://github.com/theunkn0wn1/ansible_test_submodules_subm4.git'
```

Comment:
I'm not sure if we should add more external repos for these tests to depend on. It would probably be better for these to be added to an ansible-managed org.

Reply:
If you want to move these to an ansible-managed org, please do.
I have no such access, and the existing test suite repositories lack the necessary coverage